data-raw/DATASET.R

```diff
@@ -77,20 +96,233 @@ seCombinedGeneExpected <- transcriptToGeneExpression(seCombined)
 seCombinedExtendedGeneExpected <- transcriptToGeneExpression(seCombinedExtended)
 
 
-set.seed(1234)
-seqlevelsStyle(gr) <- "UCSC"
-seUCSCExpected <- bambu(reads = test.bam, annotations = gr, genomeSequence = fa.file, extendAnnotations = FALSE)
-
-
-
-
 usethis::use_data(data1, data2, data3, data4, data5,
     estOutput_woBC,
     estOutput_wBC,
     standardJunctionModels_temp,
     seWithDistExpected,
     seGeneExpected, seExtendedGeneExpected,
     seCombinedGeneExpected, seCombinedExtendedGeneExpected,
-    seUCSCExpected,
     internal = TRUE, overwrite = TRUE
 )
+
+
+## inst data creation
+.libPaths("/usr/local/lib/R/site-library")
```

Comment:
can we remove references to paths?

Reply:
yes, we can remove this! this is more for my local usage.